hazelcast/src/main/java/com/hazelcast/nio/serialization/GenericRecord.java

```diff
@@ -43,23 +45,23 @@
  *
  *             return null;
  *         });
- * </pre>
- * Another example with EntryProcessor to demonstrate how to read, modify and set back to the map:
- * <pre>
+ * }</pre>
+ * Another example with EntryProcessor to demonstrate how to read, modify and set the value back to the map:
+ * <pre>{@code
  * map.executeOnKey("key", (EntryProcessor<Object, Object, Object>) entry -> {
  *             GenericRecord genericRecord = (GenericRecord) entry.getValue();
  *             GenericRecord modifiedGenericRecord = genericRecord.cloneWithBuilder()
  *                     .setInt("age",22).build();
  *             entry.setValue(modifiedGenericRecord);
  *             return null;
  *         });
- * </pre>
+ * }</pre>
  * <p>
- * GenericRecord also allows to read from a cluster without having the classes on the client side.
+ * GenericRecord also allows reading from the cluster without having the classes on the client side.
```

Comment:
```suggestion
 * GenericRecord also allows reading from a cluster without having the classes on the client side.
```

I think "a" is more suitable here since we are talking about any cluster.